code-of-conduct.md

```diff
@@ -2,47 +2,48 @@
 
 # Code of conduct and anti-harassment policy
 
-We expect everyone involved in our community to follow this code of conduct. This applies to all of our methods of communication.
+We, the denhac Board of Directors, expect everyone involved in our community to follow this code of conduct. This applies to all of our methods of communication.
 
-We do not tolerate harassment of people at our events or space in any form. 
+denhac does not tolerate harassment of people at our events or space in any form. 
 
 Behavior considered as harassment includes, but is not limited to:
 
 1. Offensive comments including epithets, slurs, negative stereotyping, verbal kidding, teasing, jokes, or intimidating acts related to race, religion, gender, gender identity and expression, sexual orientation, disability, or physical appearance. Consider that calling attention to differences can feel alienating.
-2. Gratuitous sexual or obscene objects, images, discussions, or behavior.
-3. Physical contact, romantic advances, or sexual attention that continues after a request to stop or "no" response.
-4. Subtle or explicit demands for sexual activity.
-5. Verbal abuse.
-6. Threats or incitement of violence towards any individual, including encouraging a person to engage in self-harm
-7. Deliberate intimidation by words, gestures, body language, or menacing behavior
-8. Stalking
-9. Harassing photography or recording, including logging online activity for harassment purposes
-10. Continued one-on-one contact or communication after requests to cease
-11. Deliberate “outing” of a sensitive aspect of a person’s identity without their consent
-12. Deliberate misgendering. This includes dead-naming or persistently using a pronoun that does not correctly reflect a person’s gender identity
-13. Retaliating against anyone who files a complaint that someone has violated this code of conduct 
+2. Gratuitous sexual or obscene objects, images, discussions, or behavior
+3. Physical contact, romantic advances, sexual attention, or other contact or messaging that continues after a request to stop or a negative/"no" response
+4. Physical contact, sexual or romantic attention, or other actions where informed and voluntary consent can be expected to be received, but was not received in advance
+5. Subtle or explicit demands for sexual or romantic activity
+6. Physical or verbal abuse
+7. Threats or incitement of violence towards any individual, including encouraging a person to engage in self-harm
+8. Deliberate intimidation by words, gestures, body language, or menacing behavior
+9. Stalking
+10. Harassing photography or recording, including logging online activity for harassment purposes
+11. Continued one-on-one contact or communication after active requests to cease, multiple instances of no response, or other indicators of a lack of consent
```

Comment:
would love feedback on how to make this more actionable / indicative of a pattern of behavior